doc/external-libs/lapack.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN"
+	  "http://www.w3.org/TR/html4/loose.dtd">
+
+<html>
+  <head>
+    <title>The deal.II Readme on interfacing to BLAS/LAPACK</title>
+    <link href="../screen.css" rel="StyleSheet">
+    <meta name="copyright" content="Copyright (C) 2010 - 2017 by the deal.II authors">
+    <meta name="keywords" content="deal.II">
+  </head>
+  <body>
+
+    <h2>Installation of <acronym>BLAS/LAPACK</acronym></h2>
+
+    <p>
+      <a href="https://netlib.org/lapack/">LAPACK</a>
+      is a library of Fortran subroutines for solving the most
+      commonly occurring problems in numerical linear algebra.
+      LAPACK is available on <a href="https://github.com/Reference-LAPACK/lapack">GitHub</a>. LAPACK releases are also available on <a href="https://netlib.org/lapack/">netlib</a>.
```

Comment:
I think it would be worth mentioning that these are standard reference implementation and that other implementations exist: OpenBLAS and MKL come to mind.

Reply:
Hi Prof. Heister,
Thanks for the feedback.
I have updated the document accordingly.